src/docs/go-grpc.yaml

```diff
@@ -0,0 +1,19 @@
+slug: go-grpc
+name: Go gRPC
+written: September 14, 2021
+updated: September 14, 2021
+protocol: gRPC
```

Comment:
Could we make this all lower case `grpc`? (it won't display to end users, just used to match technologies up for mtls)

Reply:
Name and protocol? The official name is `gRPC`

Reply:
Yeah, i've been doing these lowercase across the board for predictability since it's for machines to match:

```suggestion
protocol: grpc
```

Reply:
Fixed in 328fa8e